tests/python/relay/test_op_level2.py

```diff
@@ -127,8 +150,14 @@ def run_test_conv2d(dtype, out_dtype, scale, dshape, kshape,
                     padding=(1, 1), channels=10, kernel_size=(3 ,3))
     kshape = (10, 3, 1, 3)
     # mixed precision.
-    run_test_conv2d("int8", "int32", 1, dshape, kshape,
+    run_test_conv2d("int8", "int8", 1, dshape, kshape,
                     padding=(0, 1), channels=10, kernel_size=(1 ,3))
+    # mixed precision group conv2d
+    # NOTE(kumasento): This test cannot pass
```

Comment:
I tried to schedule a `group_conv2d_NCHW4c_int8` command here but this test failed. Needs some further investigation.

Reply:
@kumasento By default we use `direct` schedule, so this test won't pass. I would suggest removing this test for now

Reply:
Alright, I will just remove these commented lines of tests.